### DISCUSSION TITLE
Setting the IR LED on/off still requires the Wyze app?

### DISCUSSION THREAD

**Discussion by hacstest04f** [gtxaspec/wz_mini_hacks#426] [Q&A]
For those using the Wyze cam offline, do you still need to connect it to the Wyze server momentarily to set the IR LED (either on or off)?

There doesn't seem to be a setting in the configuration editor to turn the IR led on/off.  It would be cool if that's possible.  Even better if there's an API to do that.

**Comment by baudev** [comment]
You can control them manually:

To light on the red led: `echo '0' > /sys/devices/virtual/gpio/gpio38/value`

To light off the red led: `echo '1' > /sys/devices/virtual/gpio/gpio38/value`

Here is the two led gpio numbers:

| GPIO    | 38    | 39    |
|---------|-------|-------|
| Color | RED | BLUE |

(You can light on both to get a purple color)

**Reply by hacstest04f** [reply]
Oh I didn't mean the blue/red LED.  I meant the IR LED that illuminates the scene, as well as the IR cut filter.  It'd be nice if we didn't have to rely on Wyze server to be able to activate the IR illumination LED and move the cut filter up.  Right now, you have to use the official Wyze app to do that, then the v3 cam would default to that setting.  If you firewall your v3 cam off from internet, you're suck in that setting as the default until you again open up the internet to it and use the app again.

**Reply by baudev** [reply]
Oh yes, sorry, I read too fast!

No, the Wyze application is not mandatory. It is possible to manually switch the IR leds with `cmd irled on` or `cmd irled off`. It also enables the cul filter

**Comment by saxobroko** [comment]
If you enable the web server you have access to this little menu. Obviously the v3 cam only has access to the bottom controls, I haven't tried headlight.

![image](https://user-images.githubusercontent.com/18757374/209789034-70be60e2-5c2b-4a52-90b4-a243814b4143.png)
